src/map/skill.cpp

```diff
@@ -7312,12 +7553,37 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 	case KO_HAPPOKUNAI:
 	case RL_FIREDANCE:
 	case RL_R_TRIP:
+	case SJ_FULLMOONKICK: 
+	case SJ_NEWMOONKICK:
+	case SJ_SOLARBURST:
+	case SJ_STAREMPEROR:
+	case SJ_FALLINGSTAR_ATK:
 	{
 		struct status_change *sc = status_get_sc(src);
 		int starget = BL_CHAR|BL_SKILL;
 
 		if (skill_id == SR_HOWLINGOFLION)
 			starget = splash_target(src);
+		if (skill_id == SJ_NEWMOONKICK) {
+			if (tsce) {
+				status_change_end(bl, type, INVALID_TIMER);
+				clif_skill_nodamage(src, bl, skill_id, skill_lv, 1);
+				break;
+			} else
+				sc_start(src, bl, type, 100, skill_lv, skill_get_time(skill_id, skill_lv));
+		}
+		if (skill_id == SJ_STAREMPEROR && sc && sc->data[SC_DIMENSION]) {
```

Comment:
sc is not declared

Reply:
Not sure what you mean by "not declared".

Reply:
Try to recompile, it will shows sc is not declared error. I think its from sc in line 7575.

Reply:
![image](https://user-images.githubusercontent.com/11705163/76432819-74226980-63ee-11ea-8e38-d8b698a9a813.png)

Reply:
Looks like you have a merge conflict. I just checked the source and CI didn’t complain about it missing.

Reply:
Sorry, my bad.